scripts/Muon/GUI/MuonAnalysis/muon_analysis_2.py

```diff
@@ -299,8 +300,5 @@ def setup_fitting_notifier(self):
             self.dockable_plot_widget.presenter.plot_guess_observer)
 
     def closeEvent(self, event):
-        self.tabs.closeEvent(event)
-        self.dockable_plot_widget_window.widget().close()
-        self.removeDockWidget(self.dockable_plot_widget_window)
-        super(MuonAnalysisGui, self).closeEvent(event)
         self.context.ads_observer = None
+        super(MuonAnalysisGui, self).closeEvent(event)
```

Comment:
I think the removal of some of the other lines here has led to un-docked plots and tabs not closing when the main window is closed but instead hanging around in an undefined state with no obvious way to close them

Reply:
I just tried this on the build from the last PR and everything seemed to close ok

Reply:
Should be fixed now